portfolio/src/main/webapp/script.js

```diff
@@ -588,6 +587,36 @@ let Chart = class {
     }
 }
 
+function toggleChart(bool){
+    if (currentUser.active == false || !currentUser){
+        alert("you need to sign in to access the chart");
+        return;
+    }
+    setHidden("chart-wrapper", bool);
+}
+
+function populateVotes(){
+    let el;
+    let ids = ["HipHop", "Country", "Pop", "Rock", "Classical", "RandB"];
+
+    el = document.getElementById(ids[0]);
+    el.innerText = currentChartData.hMap.HipHop || 0;
```

Comment:
This can be re-written as a loop:
```
for (genreId of ids) {
  const genreDisplay = document.getElementById(genreId);
  genreDisplay.innerText = currentChartData.hMap[genreId] || 0;
}
```

Reply:
Noted. Unforunately this function didn't make the cut